mantle/kola/tests/rhcos/upgrade.go

```diff
@@ -56,10 +56,11 @@ func init() {
 		Run:         rhcosUpgradeBasic,
 		ClusterSize: 1,
 		// if renaming this, also rename the command in kolet-httpd.service below
-		Name:     "rhcos.upgrade.basic",
-		FailFast: true,
-		Tags:     []string{"upgrade"},
-		Distros:  []string{"rhcos"},
+		Name:                 "rhcos.upgrade.basic",
+		FailFast:             true,
+		Tags:                 []string{"upgrade"},
+		Distros:              []string{"rhcos"},
+		ExcludeArchitectures: []string{"aarch64"}, //upgrade test not valid for aarch64
```

Comment:
Hmm, this looks like a distinct change from the "Enable Tang for Power and ARMv8" change, no? (AFAICT, this test used to run on aarch64.) OK to keep in the same commit if you prefer, though is there a better comment or ideally link we can put here with more info why it's not valid?

Reply:
sorry, i somehow missed getting back to this! added a comment. because there is no official rhcos release, there will nothing to upgrade from. so this was excluded.